src/wfs/src/test/java/org/geoserver/wfs/BatchManagerTest.java

```diff
@@ -177,6 +177,45 @@ public void testAggregationNoDeleteAgg() throws Exception {
         testAggregationWithDeleteBatchSize(-1);
     }
 
+    @Test
+    public void testLargeBatchDeletion() {
+
+        final int batch_size_for_deletion = 40;
```

Comment:
Our coding convention uses underscores only for constants. So either:
``batchSizeForDeletion``
or
``BATCH_SIZE_FOR_DELETION`` (since it's effectively a constant)